tests/test_auxtel_enable_atcs.py

```diff
@@ -38,9 +38,19 @@ class TestEnableATTCS(
     async def basic_make_script(self, index):
         self.script = EnableATTCS(index=index)
         self.atcs_mock = ATCSMock()
+        self.script._attcs.disable_checks_for_components = unittest.mock.Mock()
```

Comment:
Same comment as above, any reason to change the unit test? they should work as is no?